sycl/test/basic_tests/handler/parallel_for_arg_restrictions.cpp

```diff
@@ -71,6 +71,18 @@ int main() {
     CGH.parallel_for(sycl::nd_range{sycl::range{1, 1, 1}, sycl::range{1, 1, 1}},
                      [=](ConvertibleFromItem<3>) {});
   });
+  Q.submit([&](sycl::handler &CGH) {
+    // expected-error@sycl/handler.hpp:* {{Kernel should be invocable with a sycl::item}}
+    CGH.parallel_for(sycl::range{1}, [=](auto &) {});
+  });
+  Q.submit([&](sycl::handler &CGH) {
+    // expected-error@sycl/handler.hpp:* {{Kernel should be invocable with a sycl::item}}
+    CGH.parallel_for(sycl::range{1, 1}, [=](auto &) {});
+  });
+  Q.submit([&](sycl::handler &CGH) {
+    // expected-error@sycl/handler.hpp:* {{Kernel should be invocable with a sycl::item}}
+    CGH.parallel_for(sycl::range{1, 1, 1}, [=](auto &) {});
+  });
```

Comment:
Please move `auto &` tests next to `auto`. If those don't exist, then we need to add them here.

Reply:
Testing `parallel_for` with an auto parameter is already tested by `handler_generic_lambda_interface.cpp` https://github.com/intel/llvm/blob/be8e031cd078ea0298a773aa5fc743c18e50fe97/sycl/test/basic_tests/handler/handler_generic_lambda_interface.cpp#L16-L25